app/scripts/modules/core/src/securityGroup/AllSecurityGroupsCtrl.js

```diff
@@ -37,10 +37,12 @@ module.exports = angular.module('spinnaker.core.securityGroup.all.controller', [
 
       app.setActiveState(app.securityGroups);
       $scope.$on('$destroy', () => {
+        app.securityGroups.deactivate();
         app.setActiveState();
         groupsUpdatedSubscription.unsubscribe();
       });
 
+      app.securityGroups.activate();
```

Comment:
should really consider moving the `activate`/`deactivate` code to the data source management itself and out of the controllers/root components